### DISCUSSION TITLE
Override the default mutations/queries

### DISCUSSION THREAD

**Discussion by suryaxan** [graphql-devise/graphql_devise#168] [Q&A] [answered]
Can someone help in understanding how to override the default mutations(say `signUp`) with our custom logic?

**Comment by 00dav00** [comment, marked as answer]
Hi @suryaxan, have you checked the dummy app used for testing?
Here you have an example of customizing the [signUp mutation](https://github.com/graphql-devise/graphql_devise/blob/master/spec/dummy/app/graphql/mutations/sign_up.rb).
